src/course-team/CourseTeam.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useIntl, injectIntl } from '@edx/frontend-platform/i18n';
+import {
+  Button,
+  Container,
+  Layout,
+  useToggle,
+} from '@edx/paragon';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { Add } from '@edx/paragon/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import SubHeader from '../generic/sub-header/SubHeader';
+import messages from './messages';
+import CourseTeamSideBar from './course-team-sidebar/CourseTeamSidebar';
+import AddUserForm from './add-user-form/AddUserForm';
+import AddTeamMember from './add-team-member/AddTeamMember';
+import CourseTeamMember from './course-team-member/CourseTeamMember';
+
```

Comment:
Let's put empty line between libraries imports and our local files, more nested first
please check it everywhere in your pr

Reply:
Done